content/blogs/2024-04-12-release-0-16.md

```diff
@@ -17,8 +17,8 @@ Let's dive into the highlights of this release.
 
 Until Kestra 0.15.11, you could configure the script tasks to run in local **processes** or in **Docker containers** by using the `runner` property.
 
-Kestra 0.16.0 introduces a new `taskRunner` property that provides more flexibility than  `runner` and allows you to deploy your code to various remote environments, including [Kubernetes](https://kestra.io/plugins/plugin-kubernetes/task-runners/runner/io.kestra.plugin.kubernetes.runner.kubernetestaskrunner), [Docker](https://kestra.io/plugins/plugin-aws/task-runners/io.kestra.plugin.scripts.runner.docker.dockertaskrunner),
- [AWS Batch](https://kestra.io/plugins/plugin-aws/task-runners/runner/io.kestra.plugin.aws.runner.awsbatchtaskrunner), [Azure Batch](https://kestra.io/plugins/plugin-azure/task-runners/runner/io.kestra.plugin.azure.runner.azurebatchtaskrunner), [Google Batch](https://kestra.io/plugins/plugin-gcp/task-runners/runner/io.kestra.plugin.gcp.runner.gcpbatchtaskrunner), and more coming in the future. Since each `taskRunner` type is a plugin, you can create your own and if you wish, you can contribute it to our open-source codebase.
+Kestra 0.16.0 introduces a new `taskRunner` property that provides more flexibility than  `runner` and allows you to deploy your code to various remote environments, including [Kubernetes](/plugins/plugin-kubernetes/task-runners/runner/io.kestra.plugin.kubernetes.runner.Kubernetes), [Docker](/plugins/plugin-aws/task-runners/io.kestra.plugin.scripts.runner.docker.Docker),
```

Comment:
it would be better to keep the old blog post introducing task runners as is, as that version uses the old naming